### ISSUE TITLE
First call to list or paragraph slow

### ISSUE THREAD

**Issue by bterlson** [domenic/ecmarkdown#19]
```
var emd = require('ecmarkdown');
var s = Date.now();
emd.paragraph('hello')
console.log("Took: " + (Date.now() - s))
```

On my machine this takes ~250ms on average, which seems excessive. Subsequent calls are very fast, however.

**Comment by domenic** [comment]
Really unsure what to do about this. I mean, I could do it at startup, increasing `require()` time :P. Might have to look in to alternatives to pegjs.

I guess I should profile if nothing else.

**Comment by domenic** [comment]
Fixed by #22. Further speed improvements possible via #12.
